src/runtime/arc.ts

```diff
@@ -351,6 +351,10 @@ ${this.activeRecipe.toString()}`;
   get activeRecipe() { return this._activeRecipe; }
   get recipes() { return this._recipes; }
 
+  get arcId() {
```

Comment:
docs and return type (string?)

Feels like there has to be a better way of getting this?

Reply:
moved it to be a data member of Arc, probably it's better.